packages/app/src/explorer/utils.ts

```diff
@@ -74,3 +74,17 @@ export function focusPrevious(e: KeyboardEvent<HTMLButtonElement>) {
     e.preventDefault();
   }
 }
+
+export function focusFirst(e: KeyboardEvent) {
```

Comment:
For some reason, the types of `@react-hookz/web` do not allow an handler taking an event of type React's `KeyboardEvent`... I had to rename this type to `ReactKeyboardEvent` to avoid shadowing the native `KeyboardEvent` that is accepted by `@react-hookz/web`.

Perhaps worth opening an issue ? :shrug:

Reply:
Given it is using `addEventListener` under the hood, I think the types are correct in fact.